rct229/rulesets/ashrae9012019/section16/section16rule2.py

```diff
@@ -0,0 +1,127 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rule_engine.rulesets import LeapYear
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.rulesets.ashrae9012019.ruleset_functions.compare_schedules import (
+    compare_schedules,
+)
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.utility_functions import find_exactly_one_schedule
+
+
+class Section16Rule2(RuleDefinitionListIndexedBase):
+    """Rule 2 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule2, self).__init__(
+            rmds_used=produce_ruleset_model_instance(
+                USER=False,
+                BASELINE_0=True,
+                PROPOSED=True,
+            ),
+            each_rule=Section16Rule2.RuleSetModelDescriptionRule(),
+            index_rmd=BASELINE_0,
+            id="16-2",
+            description="The baseline elevator motor use shall be modeled with the same schedule as the proposed design. Rule Assertion: B-RMD = P-RMD",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0]",
+            required_fields={"$": ["calendar"], "$.calendar": ["is_leap_year"]},
+            data_items={"is_leap_year_b": (BASELINE_0, "calendar/is_leap_year")},
+        )
+
+    class RuleSetModelDescriptionRule(RuleDefinitionListIndexedBase):
+        def __init__(self):
+            super(Section16Rule2.RuleSetModelDescriptionRule, self).__init__(
+                rmds_used=produce_ruleset_model_instance(
+                    USER=False, BASELINE_0=True, PROPOSED=True
+                ),
+                each_rule=Section16Rule2.RuleSetModelDescriptionRule.ElevatorRule(),
+                index_rmd=BASELINE_0,
+                list_path="buildings[*].elevators[*]",
+            )
+
+        def is_applicable(self, context, data=None):
+            rmd_p = context.PROPOSED
+
+            return find_all("$.buildings[*].elevators", rmd_p)
```

Comment:
This should be ` find_all("$.buildings[*].elevators[*]", rmd_p)`

Reply:
Good point! Addressed